targets/ath79-generic

```diff
@@ -263,11 +263,17 @@ device('tp-link-archer-d50-v1', 'tplink_archer-d50-v1', {
 
 device('tp-link-cpe210-v1', 'tplink_cpe210-v1', {
 	manifest_aliases = {
-		'tp-link-cpe210-v1.0', -- Upgrade from OpenWrt 19.07
-		'tp-link-cpe210-v1.1', -- Upgrade from OpenWrt 19.07
+		'tp-link-cpe210-v1.0', -- upgrade from OpenWrt 19.07
+		'tp-link-cpe210-v1.1', -- upgrade from OpenWrt 19.07
```

Comment:
Changing the casing here is unnecessary churn.

Reply:
fine.

Reply:
resolved in bd4408f74c322e009d863c428f14c17b73a0e978